README.md

```diff
@@ -50,6 +50,10 @@ Email address for `ds-wagtail` user with access to page content.
 
 Password address for `ds-wagtail` user with access to page content.
 
+### `TNA_SCRAPER_BASE_URL`
```

Comment:
@TNA-Allan What do you think about using `ETNA_` as a prefix for these instead, since they are only related to the Etna scrapers?

Reply:
Also the comments mentions ds-wagtail

Reply:
Okay, cool. I'm happy if you're happy :)

Reply:
This is what I thought. Hope this will be Ok (for now)